chat/src/main/java/com/qiscus/sdk/data/remote/QiscusApi.java

```diff
@@ -390,6 +396,13 @@ public static QiscusApi getInstance() {
                                                @Query("query") String query,
                                                @Query("room_id") int roomId,
                                                @Query("last_comment_id") int lastCommentId);
+
+        @FormUrlEncoded
+        @POST("/api/v2/mobile/get_rooms_info")
+        Observable<JsonElement> getRoomsInfo(@Field("token") String token,
+                                             @Field("room_id[]") List<String> roomId,
```

Comment:
ganti list of integer aja ya @adicatur soalnya room id itu int di pojonya. List<Integer> roomIds

Reply:
ok